src/parser-typescript.js

```diff
@@ -35,7 +35,10 @@ function tryParseTypeScript(text, jsx) {
     tokens: true,
     comment: true,
     useJSXTextNode: true,
-    ecmaFeatures: { jsx }
+    ecmaFeatures: { jsx },
+    // Override logger function with noop,
+    // to avoid unsupported version errors being logged
+    loggerFn: Function.prototype
```

Comment:
Can you write () => {} or function() {} instead? Function.prototype is really weird for a noop